incubator/hnc/e2e/tests/issues_test.go

```diff
@@ -7,27 +7,34 @@ import (
 var _ = Describe("Issues", func() {
 
 	const (
-		nsParent = "issues-parent"
-		nsChild = "issues-child"
+		nsParent = "parent"
+		nsChild = "child"
+		nsSub1 = "sub1"
+		nsSub2 = "sub2"
+		nsSub1Sub1 = "sub1-sub1"
+		nsSub2Sub1 = "sub2-sub1"
+		nsSubSub2 = "sub-sub2"
+		nsSubChild = "sub-child"
 	)
 
 	BeforeEach(func() {
-		cleanupNamespaces(nsParent, nsChild)
+		cleanupNamespaces(nsParent, nsChild, nsSub1, nsSub2, nsSub1Sub1, nsSub2Sub1, nsSubSub2, nsSubChild)
+		mustRun("kubectl get ns")
```

Comment:
Is this line added to confirm the namespaces are not there? If so, maybe add checks since this line itself doesn't really do anything? Or just remove this line.

Reply:
This is to prevent the cases where namespaces are not cleaned from last run. For example, one case could be the test was interrupted and terminated (i.e. you hit ctrl+c), and the AfterEach did not run.

Reply:
Oh I meant line 22. What does `k get ns` do?

Reply:
oh it was for my debugging session, forgot to remove it sorry. Gone now